ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/management-api-models.j2

```diff
@@ -0,0 +1,16 @@
+{# Management API Models #}
+{% if management_api_models is arista.avd.defined %}
+!
+management api models
+{%     for provider in management_api_models.providers | arista.avd.convert_dicts('name') | arista.avd.natural_sort('name') %}
+   !
+   provider {{ provider.name }}
+{%         for path in provider.paths | arista.avd.convert_dicts('path') | arista.avd.natural_sort('path') %}
+{%             set provider_cli = "path " ~ path.path %}
+{%             if path.enabled is arista.avd.defined(false) %}
+{%                 set provider_cli = provider_cli ~ " disabled" %}
+{%             endif %}
+      {{ provider_cli }}
+{%         endfor %}
```

Comment:
```suggestion
{%         if provider.name is arista.avd.defined and provider.name in ['smash', 'sysdb'] %}
{%             for path in provider.paths | arista.avd.convert_dicts('path') | arista.avd.natural_sort('path') %}
{%                 set provider_cli = "path " ~ path.path %}
{%                 if path.enabled is arista.avd.defined(false) %}
{%                     set provider_cli = provider_cli ~ " disabled" %}
{%                 endif %}
      {{ provider_cli }}
{%             endfor %}
{%         endif %}
```

Reply:
From Claus suggestion. I would prefer ```{%         if provider.name is defined and provider.name in ['smash', 'sysdb'] %}``` for line 8

Reply:
@ClausHolbechArista any reason to use ```arista.avd.convert_dicts('path')``` in your suggestion? I believe this is a new feature and we are adding it as a list.

Reply:
No you are correct Tony. We only use that for hitless refactoring of old data models. Thanks

Reply:
I have updated suggestion above

Reply:
👍 will update the template as suggested.